Cargo.toml

```
@@ -55,6 +55,22 @@ overflow-checks = false
 # per-crate configuration isn't specifiable in the environment.
 codegen-units = 10000
 
+# These dependencies of the standard library implement symbolication for
+# backtraces on most platforms. Their debuginfo causes both linking to be slower
+# (more data to chew through) and binaries to be larger without really all that
+# much benefit. This section turns them all to down to have no debuginfo which
+# helps to improve link times a little bit.
+[profile.release.package.miniz_oxide]
+debug = 0
+[profile.release.package.gimli]
+debug = 0
+[profile.release.package.addr2line]
+debug = 0
+[profile.release.package.object]
+debug = 0
+[profile.release.package.adler]
+debug = 0
```

Can we make it more dense?
```suggestion
[profile.release.package]
miniz_oxide = { debug = 0 }
gimli = { debug = 0 }
addr2line = { debug = 0 }
object = { debug = 0 }
adler = { debug = 0 }
```

---

This seems really unnecessary.

---

Just out of curiosity, does TOML have the Nix thing of letting you write `gimli = { debug = 0 }` as `gimli.debug = 0`?
IMO using the dot notation would feel more natural, but then again maybe that's personal bias.

---

Yes TOML supports `gimli.debug = 0`, so I've gone ahead and updated to use that.